src/types/profile.js

```diff
@@ -734,6 +734,17 @@ export type SampleUnits = {|
   +threadCPUDelta: ThreadCPUDeltaUnit,
 |};
 
+export type ExtraProfileInfoSection = {|
+  // section label
+  label: string,
+  entries: Array<{|
+    label: string,
+    format: MarkerFormatType,
```

Comment:
Just like Nazim, I'm not a big fan of reusing `MarkerFormatType` like that. Let's say we can keep it for now so that we don't block you, but in the long run we may want to generalize this in a better way.

__update after looking at the rest of the code__: as said above, my preference would be to generalize it now, unless you don't have time for this.